front/services/api/ReqwestApiClient.js

```diff
@@ -139,6 +139,25 @@ export default class ReqwestApiClient {
         });
     }
 
+    unregisterFromCitizenAction(citizenActionSlug, token, callback) {
+        let request = this._reqwest({
+            url: '/action-citoyenne/'+citizenActionSlug+'/desinscription',
+            type: 'html',
+            method: 'post',
+            data: {
+                'token': token
+            }
+        });
+
+        request.then((response) => {
+            callback(JSON.parse(response));
+        });
+
+        request.fail(() => {
+            callback(null);
```

Comment:
In that case, the page is reloaded and the user believe he unregistered successfully while it is not ?